css/_uswds-theme-custom-styles.scss

```diff
@@ -519,6 +519,19 @@ img {
   }
 }
 
+.site-figure {
+  @include u-display("flex");
+  @include u-flex("column");
+  @include u-flex("align-center");
```

Comment:
These styles follow standards, but I don't see a visual difference with these disabled. Do we need them?

Reply:
Removed in [1c0a212](https://github.com/uswds/uswds-site/pull/1751/commits/1c0a212669a3082bcd6fadae7ad5d0dd453fb859)